deploy/docker/README.md

```diff
@@ -50,48 +50,41 @@ The EMQX broker runs as Linux user `emqx` in the docker container.
 
 All EMQX Configuration in [`etc/emqx.conf`](https://github.com/emqx/emqx/blob/master/apps/emqx/etc/emqx.conf) can be configured via environment variables.
 
-By default, the environment variables with `EMQX_` prefix are mapped to key-value pairs in configuration files.
-
-You can change the prefix by overriding `HOCON_ENV_OVERRIDE_PREFIX`.
+The environment variables with `EMQX_` prefix are mapped to key-value pairs in configuration files.
 
 Example:
 
 ```bash
-EMQX_LISTENERS__SSL__DEFAULT__ACCEPTORS <--> listeners.ssl.default.acceptors
-EMQX_ZONES__DEFAULT__MQTT__MAX_PACKET_SIZE <--> zones.default.mqtt.max_packet_size
+EMQX_DASHBOARD__DEFAULT_PASSWORD       <--> dashboard.default_password
+EMQX_NODE__COOKIE                      <--> node.cookie
+EMQX_LISTENERS__SSL__default__ENABLE   <--> listeners.ssl.default.enable
```

Comment:
```suggestion
EMQX_LISTENERS__SSL__DEFAULT__ENABLE   <--> listeners.ssl.default.enable
```

Reply:
this is intentionally left lower case to demo that lower case letters also work.

Reply:
Cool, maybe add some note, it's easy to get confused